.doks/doks-cluster/Makefile

```diff
@@ -34,27 +34,34 @@ setup_vars:
 	echo "num_nodes = $(NUM_NODES)" >> $(VARS_FILE)
 	echo "cluster_name = \"$(DOKS_CLUSTER_NAME)\"">> $(VARS_FILE)
 
+.PHONY: start_cluster
+start_cluster: provision_cluster set_cluster_context helm_init_upgrade install_nginx
+
+.PHONY: provision_cluster
+provision_cluster: setup_vars create_terraformrc
+# echo '1' | $(TF) init -input=false -reconfigure
```

Comment:
can you write line before this saying why this is commented plz?
Like, "Currently the tf state is not in sync with the infrastructure, we need to tweak it before we can have CD for this"